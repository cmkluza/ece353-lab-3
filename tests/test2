./sim-mips -b 1 1 1 ./tests/asm/test2.s ./tests/out/test2.out
./tests/sols/test2.sol