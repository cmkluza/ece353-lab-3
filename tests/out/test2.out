program name: ./tests/asm/test2.s
stage utilization: 0.739130  0.695652  0.869565  0.000000  0.826087 
register values 0  0  0  0  0  0  0  1  2  3  255  256  32767  0  0  3  6  65534  4  6  512  -262140  1  -254  1  0  0  0  0  0  0  68
